### ISSUE TITLE
Error on creating ecs-fargate with mount_point is null

### ISSUE THREAD

**Issue by chitopunk** [cn-terraform/terraform-aws-ecs-fargate#21]
```terraform
Error: Invalid function argument

  on .terraform/modules/sonar.ecs_fargate.td.container_definition/main.tf line 17, in locals:
  17:   mount_points = length(var.mount_points) > 0 ? [
    |----------------
    | var.mount_points is null
```

**Comment by jnonino** [comment]
Released in version [2.0.21](https://registry.terraform.io/modules/cn-terraform/ecs-fargate/aws/2.0.21)

**Comment by jnonino** [comment]
Thanks @chitopunk !!!
